.travis.yml

```diff
@@ -3,7 +3,9 @@ cache:
   bundler: true
   directories:
     - node_modules
-rvm: 2.1.5
+rvm:
+  - 2.1.6
```

Comment:
I'm a sucker for the latest and greatest! :)
We should be able to test it out with 2.2.2 fairly easily. Samson is a relatively small tool and we should be able to roll it back if we run into trouble. We looking to update zendesk_voyager repo to 2.2.2 soon as well.